tests/test_async_processing_import_export.py

```diff
@@ -598,7 +598,7 @@ def test_stac_source_filter_error_import(self):
 
     def test_raster_import_resample_resolution(self):
         """
-        Code test raster import with set resampling and resolution with http reponse 200
+        Test raster import by setting resampling and resolution, with http response 200
```

Comment:
I'd rather keep the `with set resampling an resolution` here and in the following tests. Maybe change it to `defined resampling and resolution`, to make clear these values are set/defined during the import?

Reply:
OK, I have reworded it in 73839ed.

Reply:
Looks good to me.